qutebrowser/browser/greasemonkey.py

```diff
@@ -108,13 +109,18 @@ def code(self):
         browser's debugger/inspector will not match up to the line
         numbers in the source script directly.
         """
+        # Don't use Proxy on this webkit version, the support isn't there.
+        use_proxy = not (
+            objects.backend == usertypes.Backend.QtWebKit and
+            version.qWebKitVersion() == '602.1')
```

Comment:
This is failing under test in an environment that doesn't have webkit installed because `test/conftest.py::set_backend()` is setting `objects.backend` to `usertypes.Backend.QtWebKit` by default so the second clause is being executed but `version.qWebKitVersion` is `None`. Should I change it like so?
```diff
diff --git i/tests/conftest.py w/tests/conftest.py
index 97d5623372a9..5043abb4dd5d 100644
--- i/tests/conftest.py
+++ w/tests/conftest.py
@@ -223,8 +223,10 @@ def check_display(request):
 @pytest.fixture(autouse=True)
 def set_backend(monkeypatch, request):
     """Make sure the backend global is set."""
-    backend = (usertypes.Backend.QtWebEngine if request.config.webengine
-               else usertypes.Backend.QtWebKit)
+    if not request.config.webengine and version.qWebKitVersion:
+        backend = usertypes.Backend.QtWebKit
+    else:
+        backend = usertypes.Backend.QtWebEngine
     monkeypatch.setattr(objects, 'backend', backend)
 
```